payroll/views.py

```diff
@@ -130,7 +130,7 @@ def get_context_data(self, **kwargs):
             source_link = None
 
         context.update({
-            'jobs': Job.of_employer(self.object.id, n=5),
+            'salaries': Salary.of_employer(self.object.id, n=5),
```

Comment:
One last thing: The Person view also uses the person table template. So, you'll need to update the context variable for that view, as well.